gradle.properties

```diff
@@ -5,3 +5,4 @@ org.gradle.vfs.watch=true
 org.gradle.configuration-cache=true
 org.gradle.configureondemand=true
 org.gradle.parallel=true
+VERSION_NAME=2023.09.18.135915-444f476-SNAPSHOT
```

Comment:
Probably need to remove - this is for testing

Reply:
Done